views/js/controller/creator/helpers/changeTracker.js

```diff
@@ -0,0 +1,243 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 Open Assessment Technologies SA ;
+ */
+/**
+ * Track the change within the testCreator
+ * @author Juan Luis Gutierrez Dos Santos <juanluis.gutierrezdossantos@taotesting.com>
+ */
+define([
+    'jquery',
+    'lodash',
+    'i18n',
+    'lib/uuid',
+    'core/eventifier',
+    'ui/dialog'
+], function (
+    $,
+    _,
+    __,
+    uuid,
+    eventifier,
+    dialog
+) {
+    'use strict';
```

Comment:
is `'use strict'`necessary? According to the [FE coding style guides](https://oat-sa.atlassian.net/wiki/spaces/DEV/pages/79037064/TAO+Frontend+Coding+guide) strict mode should not be used on ES6 modules (because ES6 modules and classes are [always ](http://www.ecma-international.org/ecma-262/6.0/#sec-strict-mode-code)on strict mode by default).

Reply:
When we remove that, @jsconan tell us to don't do. We can have a discussion again, but probably don't change in this PR